### PR TITLE
Wallet Connector Improvements

### PR DESCRIPTION
**Pull request by mayankchhabra**
Changes:

- Simplified Tor installation method for Windows users.
- Note on Specter Desktop instructions to differentiate between Umbrel and native Specter apps.
- Responsive enlarged QR size on mobile devices.
- Use just "Other" category for protocols instead of "Wallets" and "Protocols" in wallet selection dropdown ('cause users might not understand the meaning of "Protocols".) 
- Some minor design and wording tweaks.

---

Also figured out how to disable auto-lint on save in VSCode so no more linting noise! 🎉

### CODE REVIEW

File: src/components/ConnectWallet/Wallets/ElectrumDesktop.vue [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
+        <input-copy :value="`${urls.bitcoin.electrum.address}:${urls.bitcoin.electrum.port}:t`" auto-width></input-copy>
       </step>
       <step>
         Go to the <span class="font-weight-bold">"Proxy"</span> tab.
       </step>
       <step>
         Check <span class="font-weight-bold">"Use Tor proxy at port 9050"</span>
-        or manually enter this:
-        <input-copy value="127.0.0.1" auto-width></input-copy>
-        <input-copy value="9050" auto-width></input-copy>
```
Comment (louneskmt) [inline src/components/ConnectWallet/Wallets/ElectrumDesktop.vue:26]: I think we should let these instructions as I've seen cases where Electrum doesn't detect Tor running on port 9050.

File: src/components/ConnectWallet/Wallets/Sparrow.vue [hunk 2]
```diff
@@ -21,10 +24,6 @@
       <step>
         After verifying, click <span class="font-weight-bold">"Close"</span>.
       </step>
-      <step>
-        At any moment, you can turn on or off the connection using the switch 
-        in the bottom right of the window.
-      </step>
```
Comment (louneskmt) [inline src/components/ConnectWallet/Wallets/Sparrow.vue:27]: Why remove this? Maybe rephrase it? 

Thinking about it, maybe Sparrow already says on first run?

File: src/components/ConnectWallet/Wallets/ElectrumDesktop.vue (hunk 1 above)
Comment (mayankchhabra) [reply src/components/ConnectWallet/Wallets/ElectrumDesktop.vue:26]: Are you sure about that? I thought it always provided that option if it found Tor on port 9050 or 9150. And the only time it didn't show the option was when it was unable to find Tor... in which case manually entering the details won't help?

File: src/components/ConnectWallet/Wallets/Sparrow.vue (hunk 2 above)
Comment (mayankchhabra) [reply src/components/ConnectWallet/Wallets/Sparrow.vue:27]: Because it's a wallet connector and not a wallet disconnector 😝

Haha, but seriously I don't think we need to add it because:

1. The purpose of wallet connector is to help user connect their wallet to Umbrel, so we don't need to add steps that explain how to disconnect from Umbrel.
2. Consistency: we don't have any such step that explains how to disconnect for any other wallet, so we don't need to add it here too.

File: src/components/ConnectWallet/Wallets/ElectrumDesktop.vue (hunk 1 above)
Comment (louneskmt) [reply src/components/ConnectWallet/Wallets/ElectrumDesktop.vue:26]: If you say so 👌

File: src/components/ConnectWallet/Wallets/Sparrow.vue (hunk 2 above)
Comment (louneskmt) [reply src/components/ConnectWallet/Wallets/Sparrow.vue:27]: I'm not adding this here to tell the user how to disconnect the node, but it might help users to troubleshoot their Sparrow not connecting to Umbrel. It is possible to click on this switch by mistake and disconnect the external node.
